app/code/Magento/Quote/Setup/UpgradeSchema.php

```diff
@@ -99,6 +99,38 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                 ]
             );
         }
+        if (version_compare($context->getVersion(), '2.0.7', '<')) {
+            $connection = $setup->getConnection(self::$connectionName);
+            $connection->modifyColumn(
+                $setup->getTable('quote_address', self::$connectionName),
+                'telephone',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'length' => 255,
+                ]
+            )->modifyColumn(
+                $setup->getTable('quote_address', self::$connectionName),
+                'fax',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'length' => 255,
+                ]
+            )->modifyColumn(
+                $setup->getTable('quote_address', self::$connectionName),
+                'region',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'length' => 255,
+                ]
+            )->modifyColumn(
+                $setup->getTable('quote_address', self::$connectionName),
+                'city',
+                [
+                    'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
+                    'length' => 255,
```

Comment:
Currently this method is not passing PHPMD:
```
1) Magento\Test\Php\LiveCodeTest::testCodeMess
PHP Code Mess has found error(s):
/home/travis/build/magento/magento2/app/code/Magento/Quote/Setup/UpgradeSchema.php:25	The method upgrade() has 111 lines of code. Current threshold is set to 100. Avoid really long methods.
```

Please try to reformet method in more compact manner, import `\Magento\Framework\DB\Ddl\Table` (`Alt + Enter` in PhpStorm) and you can easily fit `['type => ..., 'length' => ...]` in one line and make it more readable.

Feel free to reformat lines not introduced in your PR, if polishing up will be not enough to fit into 100 lines - refactor contents of `if (version_compare($context->getVersion(), '2.0.7', '<')) {` into a private method.

Please use a commit amend + force push.